client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss

```diff
@@ -29,22 +29,30 @@ $acquire-intent-vertical-transition-algorithm: ease-in-out;
 	}
 }
 
+.acquire-intent__footer {
+	transition: opacity $acquire-intent-transition-duration $acquire-intent-transition-algorithm;
+
+	&--hidden {
+		opacity: 0;
+		visibility: hidden;
```

Comment:
I think one of these is sufficient, if we're not transitioning, visibility alone would do.

Reply:
The transition on opacity to achieve fade in effect is declared above. But in this case we can remove `visibility: hidden` since the button is disabled so not interactive 👍